lib/uktt.rb

```diff
@@ -26,29 +26,29 @@ class Error < StandardError; end
 
   # Configuration defaults
   @config = {
-              host: Uktt::Http.api_host, 
-              version: Uktt::Http.spec_version, 
-              debug: false,
-              format: 'ostruct',
-              currency: PARENT_CURRENCY,
-              query: {},
-            }
+    host: Uktt::Http.api_host,
+    version: Uktt::Http.spec_version,
+    debug: false,
+    format: 'ostruct',
+    currency: PARENT_CURRENCY,
+    query: {},
+  }
 
   @valid_config_keys = @config.keys
 
   # Configure through hash
   def self.configure(opts = {})
-    opts.each {|k,v| @config[k.to_sym] = v if @valid_config_keys.include? k.to_sym}
+    opts.each { |k, v| @config[k.to_sym] = v if @valid_config_keys.include? k.to_sym }
   end
 
   # Configure through yaml file
   def self.configure_with(path_to_yaml_file)
     begin
-      config = YAML::load(IO.read(path_to_yaml_file))
+      config = YAML.load(IO.read(path_to_yaml_file))
     rescue Errno::ENOENT
       log(:warning, "YAML configuration file couldn't be found. Using defaults."); return
```

Comment:
How about these quotes? For consistency would be nice to make them single quotes as well.

Reply:
It didn't automatically change because of the ' in `couldn't`